version_test.go

```diff
@@ -662,3 +662,58 @@ func TestScan(t *testing.T) {
 	err = v.Scan(123) // or any other type
 	require.ErrorContains(t, err, "cannot convert int to Version")
 }
+
+func TestIncPatch(t *testing.T) {
+	testCases := []struct {
+		currentVersion string
+		nextVersion    string
+	}{
+		{"v21.2.0-alpha.1", "v21.2.1"},
+		{"v21.1.0-beta.3", "v21.1.1"},
+		{"v21.1.0-rc.3", "v21.1.1"},
```

Comment:
nit question: Should these cases be 21.1.0 exactly? that is a later version than any 21.1.0 prerelease (including cloudonly)

Or maybe IncPatch should return an error if called on a pre-release, if it's not obvious what the correct user intent is?

Reply:
I think it'd be better to make this for stable versions only... Let me rework this.

Reply:
Done